lib/extract-svg-metadata.js

```diff
@@ -1,23 +1,37 @@
-const createConfig = require('svgo/lib/svgo/config');
-const convertSVG = require('svgo/lib/svgo/svg2js');
-const createSVG = require('svgo/lib/svgo/js2svg');
-const applyPlugins = require('svgo/lib/svgo/plugins');
+const { resolvePluginConfig } = require('svgo/lib/svgo/config');
+const { parseSvg } = require('svgo/lib/parser');
+const { stringifySvg } = require('svgo/lib/stringifier');
+const { invokePlugins } = require('svgo/lib/svgo/plugins');
 const computePathBounds = require('svg-boundings').path;
 
 const metadataPrefix = 'mapbox-';
 
+// Removes `style` attribute. Otherwise, transforms will not apply. 
+// Refer to https://github.com/svg/svgo/blob/a9834efa1603198ae606765292215462ae193d0d/plugins/applyTransforms.js#L55-L57
+const removeStyleAttribute = {
```

Comment:
In the latest version of svgo, transforms are skipped if `style` attribute exists in the element. For example, the rotation will not apply in the following `path` element (causing [these tests](https://github.com/mapbox/spritezero/blob/a12d7387355a560582fd35e1dd44ad478f284f9f/test/metadata.test.js#L47,L75) fail)
```
<path style="fill:none" mapbox:metadata="stretch-x" d="M6 11H12V12H6z" transform="rotate(45 7.657 8.828)"/>
```